standards/bulk.md

```diff
@@ -0,0 +1,134 @@
+# Bulk Operations
+
+## Overview
+
+The concept of bulk operations on RESTful endpoints is quite foreign, in the sense that it is not a common practice. However, there are some use cases where it makes sense to allow a client to perform multiple operations in a single request. For example, a client may want to create multiple entities at once, or update multiple entities at once. In these cases, it is often more efficient to allow the client to perform these operations in a single request, rather than making multiple requests. However, before adding support for bulk operations you should think twice if this feature is really needed. Often network performance is not what limits request throughput. Similarly, sending multiple in-parallel requests for standard REST endpoints can be a fine solution for some clients.
+
+### Synchronous
+
+Bulk operations **MUST** be synchronous when applied to an existing resource. This means that the client will receive a response to the bulk request only after all operations have been completed. This is in contrast to asynchronous operations, where the client receives a response immediately after the request is received, and then must poll for the result of the operation. Asynchronous bulk operations are applied to new resources specifically for bulk or import to enable subsequent status updates through additional endpoints.
```

Comment:
Why are asynchronous requests for existing entities prohibited? Consider scenarios where the user doesn't need to wait like archiving documents.

Reply:
That would be part of async operations against an inherently different resource that would enable polling or webhooks.

Reply:
Why is asynchronous allowed for creating resources but prohibited for existing resources? I don't understand why they are treated differently.

Reply:
I'm not sure I understand why they wouldn't be treated differently. Are you proposing in general that all resources can be interacted with ASYNC or SYNC?

Reply:
I suggest allowing async and/or sync for all bulk operations. It may make sense to only support one or the other but I don't see why async update should be prohibited.

Reply:
I'm gathering further details first on expectations for ASYNC pattern which will affect the answer here. Currently, though the expectation is that this is NOT the pattern to be used for async communication.